disdat/api.py

```diff
@@ -874,40 +873,57 @@ def add(local_context, bundle_name, path, tags=None, treat_file_as_bundle=False)
                                                                      bundle_name,
                                                                      local_context))
 
-    # Ensure path exists before creating bundle
-    assert os.path.exists(path), "Disdat cannot find file at path: {}".format(path)
+    # The user can pass either a path or an iterable of paths,
+    # convert single paths to iterable for compatibility
+    paths = path
+    try:
+        assert os.path.exists(paths)
+        paths = [paths]
+    except TypeError:
+        pass
 
     with Bundle(local_context, bundle_name, getpass.getuser()) as b:
-        if treat_file_as_bundle:
-            # Make sure file is .csv or .tsv
-            assert str(path).endswith(('.csv', '.tsv')), 'Disdat can only add tsv/csv files as bundles, please ' \
-                                                             'provide a .csv or .tsv file.'
 
-            bundle_df = pd.read_csv(path, sep=None)  # sep=None means python parse engine detects sep
-            b.add_data(bundle_df)
+        file_list = []
+        for path in paths:
+
+            # Ensure path exists before creating bundle
+            assert os.path.exists(path), "Disdat cannot find file at path: {}".format(path)
+
+            if treat_file_as_bundle:
+                # Make sure file is .csv or .tsv
+                assert str(path).endswith(('.csv', '.tsv')), 'Disdat can only add tsv/csv files as bundles, please ' \
+                                                                 'provide a .csv or .tsv file.'
+
+                bundle_df = pd.read_csv(path, sep=None)  # sep=None means python parse engine detects sep
+                file_list.append(bundle_df)
```

Comment:
It's not a file at this point -- this treated the data in the csv as a bundle with presentation dataframe.